src/GrpAb/GrpAbFinGen.jl

```diff
@@ -662,6 +692,27 @@ function direct_sum(G::FinGenAbGroup...; task::Symbol = :sum, kwargs...)
   return _direct_product(:sum, G...; task = task, kwargs...)
 end
 
+@doc raw"""
+    direct_sum(G::FinGenAbGroup, H::FinGenAbGroup, V::Vector{<:Map{FinGenAbGroup, FinGenAbGroup}})
+
+For groups `G = prod G_i` and `H = prod H_i` as well as maps `V_i: G_i -> H_i`,
+build the induced map from `G -> H`.
+"""
+function direct_sum(G::FinGenAbGroup, H::FinGenAbGroup, V::Vector{<:Map{FinGenAbGroup, FinGenAbGroup}})
```

Comment:
This one is already added in https://github.com/thofma/Hecke.jl/pull/1488 under the name `hom_direct_sum`. Once https://github.com/thofma/Hecke.jl/pull/1488 gets merged, this should get removed here.